worker/socialPoster.js

```diff
@@ -52,7 +53,7 @@ async function postToNostr ({ message }) {
     await nostr.publish({
       created_at: Math.floor(new Date().getTime() / 1000),
       content: message,
-      tags: [],
+      tags: nostrPubkey ? [['p', nostrPubkey]] : [],
       kind: 1
     }, {
```

Comment:
You are _tagging_ the user, but we want to _mention_ them like in [this note](https://njump.me/nevent1qvzqqqqqqypzpwp69zm7fewjp0vkp306adnzt7249ytxhz7mq3w5yc629u6er9zsqqsrydfdngwm8aj7fd6xsex4zltqf65h8elg9wt3dc9wsah623799ucx8yxam):

![2025-04-15-004720_628x345_scrot](https://github.com/user-attachments/assets/f76b80e2-b98e-445a-a950-9b1762c2a927)

Reply:
This is non-trivial. Would need a way to get the user's `nprof` and all we have is the `npub`. Not sure how to decode this from the `npub` (I've had a look through the NDK docs and the useProfile hook doesn't have it). alternatively, we could add a hyperlink to their `npub` on njump. I can see that's the platform used on this project.

Reply:
I think this should do it: https://njump.me/nevent1qqs8c4kkqxdzftyz9u2pgg83tu6y04c9e4hvdecgjvyjcwy9wv83djcpz3mhxue69uhhyetvv9ujuerpd46hxtnfdupzpxlk8pemdxnklxhr42r2zh6ffvz9y80ap5099x62ynh9s54l2vkdk3z7n3 @ekzyis